src/EnergyPlus/SolarShading.cc

```diff
@@ -6385,7 +6368,7 @@ namespace SolarShading {
                         if (pssas[Surface(bkSurfNum).PenumbraID] > 0) {
                             ++JBKS;
                             BackSurfaces(TS, iHour, JBKS, HTSS) = bkSurfNum;
-                            OverlapArea = pssas[Surface(bkSurfNum).PenumbraID]/CTHETA(HTSS);
+                            Real64 OverlapArea = pssas[Surface(bkSurfNum).PenumbraID]/CTHETA(HTSS);
```

Comment:
I'm sure the compiler can tell that this is a constant, and this is a ridiculously small example, but just saying it's fine to label things that you intentionally want to enforce const-ness as const.

Reply:
Yeah, I forget to think about `const`. Can add that if there's any other changes.